pr_agent/settings/configuration.toml

```diff
@@ -33,10 +33,30 @@ add_original_user_description=false
 keep_original_user_title=false
 use_bullet_points=true
 extra_instructions = ""
+
 # markers
 use_description_markers=false
 include_generated_by_header=true
 
+[custom_labels."Bug fix"]
+description = "Fixes a bug in the code"
+[custom_labels."Tests"]
+description = "Adds or modifies tests"
+[custom_labels."Bug fix with tests"]
+description = "Fixes a bug in the code and adds or modifies tests"
+[custom_labels."Refactoring"]
+description = "Refactors the code without changing its functionality"
+[custom_labels."Enhancement"]
+description = "Adds new features or functionality"
+[custom_labels."Documentation"]
+description = "Adds or modifies documentation"
+[custom_labels."SQL modifications"]
+description = "Adds or modifies SQL queries"
```

Comment:
this seems like overfit to SQL

Reply:
I will remove the SQL version ( it should not be part of the default labels)